src/frontend/planner_test/tests/testdata/output/explain_dot_format.yaml

```
@@ -0,0 +1,45 @@
+# This file is automatically generated. See `src/frontend/planner_test/README.md` for more information.
+- name: test dot output format (logical)
+  sql: |
+    CREATE TABLE t (v1 int);
+    explain (logical, format dot) SELECT approx_percentile(0.5) WITHIN GROUP (order by v1) from t;
+  explain_output: |
+    digraph {
+        0 [ label = "LogicalAgg fields: {\l  \"aggs\": [\l    \"approx_percentile($expr1)\"\l  ]\l}" ]
+        1 [ label = "LogicalProject fields: {\l  \"exprs\": [\l    \"t.v1::Float64 as $expr1\"\l  ]\l}" ]
+        2 [ label = "LogicalScan fields: {\l  \"columns\": [\l    \"v1\"\l  ],\l  \"table\": \"t\"\l}" ]
```

You probably just need `\l` after the last curly brace to align it
```suggestion
        2 [ label = "LogicalScan fields: {\l  \"columns\": [\l    \"v1\"\l  ],\l  \"table\": \"t\"\l}\l" ]
```